plugins/outputs/timestream/timestream.go

```diff
@@ -114,11 +114,11 @@ func (*Timestream) SampleConfig() string {
 
 func (t *Timestream) Connect() error {
 	if t.DatabaseName == "" {
-		return errors.New("DatabaseName key is required")
+		return errors.New("`database_name` key is required")
```

Comment:
Please decide on which quotes to use. I would use single-quotes instead of backticks...

Reply:
Single quotes!